### PR TITLE
New files added, for contravariant-superinterface-2018

### PR DESCRIPTION
**Pull request by eernstg**
Introduce documents in `accepted/future-releases` for feature #39.

### CODE REVIEW

File: accepted/future-releases/contravariant-superinterface-2018/implementation-plan.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Implementation Plan for the Q1 2019 Superinterface Contravariance Error.
+
+Relevant documents:
+ - [Tracking issue](https://github.com/dart-lang/language/issues/113)
+ - [Feature specification](https://github.com/dart-lang/language/blob/master/accepted/future-releases/contravariant-superinterface-2018/feature-specification.md)
+
+
+## Implementation and Release plan
```
Comment (leafpetersen) [inline accepted/future-releases/contravariant-superinterface-2018/implementation-plan.md]: I think we should use an experiment flag, even though breakage chances are low.  It makes sure we roll out consistently.  Maybe "covariant-only-superinterfaces"?

Comment (mit-mit) [reply accepted/future-releases/contravariant-superinterface-2018/implementation-plan.md]: +1 to using that flag name

**Comment by mit-mit** [comment]
LGTM if the flag mentioned above is added to the spec

File: accepted/future-releases/contravariant-superinterface-2018/implementation-plan.md (hunk 1 above)
Comment (eernstg) [reply accepted/future-releases/contravariant-superinterface-2018/implementation-plan.md]: Added flag to plan.

**Pull request merged**